java/src/main/java/com/logicalclocks/hsfs/engine/SparkEngine.java

```diff
@@ -93,6 +95,53 @@ private SparkEngine() {
     sparkSession.conf().set("hive.exec.dynamic.partition.mode", "nonstrict");
     // force Spark to fallback to using the Hive Serde to read Hudi COPY_ON_WRITE tables
     sparkSession.conf().set("spark.sql.hive.convertMetastoreParquet", "false");
+
+    validateSparkConfiguration();
+  }
+
+  @SneakyThrows
```

Comment:
why did you choose not to declare the `throws`?

Reply:
because it would require added "throws FeatureStoreException" to quite a few methods and thought that it would be cleaner like this.